gcs/CONFIGURATION.md

```diff
@@ -36,7 +36,14 @@
 *   `fs.gs.delegation.token.binding` (not set by default)
 
     Delegation Token binding class.
+*   `fs.gs.delegation.token.instantiation_strategy` (default `INSTANCE_PER_SERVICE`)
```

Comment:
Looking at other properties - we do not use `_` in the property name, so probably better to be consistent and rename this property to `fs.gs.delegation.token.instantiation.strategy`

Reply:
done